streams/src/test/java/org/apache/kafka/streams/processor/internals/AssignedTasksTest.java

```diff
@@ -155,7 +154,7 @@ public void shouldSuspendRunningTasks() {
         mockRunningTaskSuspension();
         EasyMock.replay(t1);
 
-        assertThat(suspendTask(), nullValue());
+        suspendTask();
```

Comment:
Could we check that `close()` was not called, i.e. no exceptions get thrown in easy mock then? Ditto below.

Reply:
Is already covered via `EasyMock.verify(t1)` at the end of the test